client/src/components/History/ContentOperations.vue

```diff
@@ -59,90 +60,72 @@
-                        <span v-if="numSelected">{{ numSelected }} items</span>
-                        <span v-else>(Disabled)</span>
-                    </b-dropdown-item>
-                </b-dropdown>
-
-                <b-dropdown
-                    class="history-contents-list-action-menu-btn"
-                    size="sm"
-                    text="History"
-                    :disabled="!totalMatches"
-                >
-                    <b-dropdown-text id="history-op-all-content">
-                        <span v-localize>With entire history...</span>
-                    </b-dropdown-text>
-
-                    <b-dropdown-item v-b-modal:show-all-hidden-content aria-describedby="history-op-all-content">
-                        <span v-localize>Unhide All Hidden Content</span>
-                    </b-dropdown-item>
-
-                    <b-dropdown-item v-b-modal:delete-all-hidden-content aria-describedby="history-op-all-content">
-                        <span v-localize>Delete All Hidden Content</span>
                     </b-dropdown-item>
 
-                    <b-dropdown-item v-b-modal:purge-all-deleted-content aria-describedby="history-op-all-content">
-                        <span v-localize>Purge All Hidden Content</span>
-                    </b-dropdown-item>
-                </b-dropdown>
-
-                <b-dropdown
-                    class="history-contents-list-action-menu-btn"
-                    size="sm"
-                    text="Create New Content"
-                    :disabled="!totalMatches"
-                    data-description="new content menu"
-                >
-                    <template v-slot:button-content>
-                        <Icon icon="plus" />
-                        <span class="sr-only" v-localize>Create New Content</span>
-                    </template>
-
-                    <b-dropdown-text id="history-op-new-content">
-                        <span>Create new content</span>
-                    </b-dropdown-text>
-
                     <b-dropdown-item
-                        aria-describedby="history-op-new-content"
-                        @click="iframeRedirect('/dataset/copy_datasets')"
-                        data-description="copy datasets"
-                    >
-                        <span v-localize>Copy Datasets</span>
-                    </b-dropdown-item>
-
-                    <b-dropdown-item
-                        aria-describedby="history-op-new-content"
+                        aria-describedby="history-op-selected-content"
                         @click="buildDatasetList"
                         data-description="build list"
                     >
                         <span v-localize>Build Dataset List</span>
                     </b-dropdown-item>
 
                     <b-dropdown-item
-                        aria-describedby="history-op-new-content"
+                        aria-describedby="history-op-selected-content"
```

Comment:
Oh, I thought you would completely remove it :sweat_smile:
We don't need this attribute, do we? As far as I understood, we need it only if we used a _clean_ bootstrap repo https://getbootstrap.com/docs/4.6/components/dropdowns/

But since we used `<b-dropdown>` component by https://bootstrap-vue.org/docs/components/dropdown, the component does things for us. :sweat_smile:

Reply:
Ahh, I assumed Mason had added those everywhere even in b-dropdown-items for a reason.  Let me check.  If we can drop them completely, then I'll do that.

Reply:
Actually while reading docs, it says that it's recommended to assign ids to headers for accessibility. So sorry that was a right thing to do. Thanks @dannon